source/main/commands/force-open.ts

```diff
@@ -26,12 +26,15 @@ export default class ForceOpen extends ZettlrCommand {
   }
 
   /**
-    * Removes a file.
+    * Force-Opens a file, after click on internal link
     * @param {String} evt The event name
-    * @param  {Object} arg the parameters of the file to be deleted
-    * @return {Boolean} Whether the file was successfully deleted.
+    * @param  {Object} payload the parameters of the file to be opened
+    * @return {Boolean} Whether the file was successfully opened.
     */
-  async run (evt: string, arg: any): Promise<void> {
+  async run (evt: string, payload: any): Promise<void> {
+    const arg = payload.linkContents
```

Comment:
A potentially cool idea would be to use a spread operator here: `const { linkContents, newTab } = payload`

Also, this would make `arg` a tad more expressive; it was a stupid idea by me to name this variable like that 😄